### PR TITLE
Update setup-guides-for-microsoft-365.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the setup guides with the recent changes made in the admin center.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7297

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The setup guides are accessible from the [Setup guidance](https://aka.ms/setupgu
 
    ![Training & guides card in the Microsoft 365 admin center](../media/setup-guides-for-microsoft-365/adminportal-trainingandguides.png)
 
-3. Select **Step-by-step guides**.
+3. Select **Advanced deployment guides** and clikc on **All guides**
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
3. Select **Advanced deployment guides** and then select **All guides**.
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 2]
```diff
@@ -92,11 +86,19 @@ Microsoft Search helps your organization find what they need to complete what th
...
+### Configure multifactor authentication (MFA)
 
-## Guides for authentication and access
+The [Configure multifactor authentication (MFA) guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/featureexplorer/security/ConditionalAccess) provides information to Secure your organization against breaches due to lost or stolen credentials. MFA immediately increases account security by prompting for multiple forms of verification to prove a user's identity when they sign in to an app or other company resource. This prompt could be to enter a code on the user's mobile device or to provide a fingerprint scan. MFA is enabled through Conditional Access, security defaults, or per-user MFA. This guide will provide the recommended MFA option for your org, based on your licenses and existing configuration.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
The [Configure multifactor authentication (MFA) guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/featureexplorer/security/ConditionalAccess) provides information to secure your organization against breaches due to lost or stolen credentials. MFA immediately increases account security by prompting for multiple forms of verification to prove a user's identity when they sign in to an app or other company resource. This prompt could be to enter a code on the user's mobile device or to provide a fingerprint scan. MFA is enabled through Conditional Access, security defaults, or per-user MFA. This guide will provide the recommended MFA option for your org, based on your licenses and existing configuration.
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 3]
```diff
@@ -92,11 +86,19 @@ Microsoft Search helps your organization find what they need to complete what th
...
+
+### Identity security for Teams
+
+The [Identity security for teams guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/teamsidentity) helps you with some basic security steps you can take to ensure your users are safe and have the most productive time using **Teams**.
+
+### Add or sync users to Microsoft 365
+
+[This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/modernonboarding/identitywizard) will help streamline the process of getting your user accounts setup in **Microsoft 365**. Based on your environment and needs, you can choose to add users individually, migrate your on-premises directory with Azure AD cloud sync or Azure AD Connect, or troubleshoot existing sync problems when necessary.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
[This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/modernonboarding/identitywizard) will help streamline the process of getting your user accounts set up in **Microsoft 365**. Based on your environment and needs, you can choose to add users individually, migrate your on-premises directory with Azure AD cloud sync or Azure AD Connect, or troubleshoot existing sync problems when necessary.
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 4]
```diff
@@ -118,6 +120,11 @@ Upgrade to an alternative sign-in approach that allows users to access their dev
 
 Use the [Plan your passwordless deployment](https://aka.ms/passwordlesssetup) guide to discover the best passwordless authentication methods to use and receive guidance on how to deploy them.
 
+### Integrate a third-party cloud app with Azure AD
+
+[This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/azureadappintegration) helps helps IT admins to select and configure the App.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md:125]:
```suggestion
[This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/azureadappintegration) helps IT admins to select and configure the App.
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 5]
```diff
@@ -118,6 +120,11 @@ Upgrade to an alternative sign-in approach that allows users to access their dev
...
+
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
[This guide](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/azureadappintegration) helps IT admins to select and configure the App.
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 6]
```diff
@@ -130,6 +137,11 @@ The [AD FS deployment advisor](https://aka.ms/adfsguidance) provides you with st
 
 ## Guides for security and compliance
 
+### Security Analyzer
+
+The Microsoft security suite offers an integrated ecosystem to help protect your organization. This guide helps you analyze your security posture. The Microsoft secure score is a measurement of an org's security posture, with a higher number indicating a stronger security posture. The Analyzer 
+
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
The Microsoft security suite offers an integrated ecosystem to help protect your organization. This guide helps you analyze your security posture. The Microsoft secure score is a measurement of an organization's security posture, with a higher number indicating a stronger security posture.
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 7]
```diff
@@ -184,9 +196,9 @@ The [Microsoft Defender for Cloud Apps setup guide](https://aka.ms/cloudappsecur
...
 
-### Microsoft 365 Apps deployment advisor
+### Microsoft 365 Apps setup guide
 
-The [Microsoft 365 Apps deployment advisor](https://aka.ms/OPPquickstartguide) helps you get your users' devices running the latest version of Office products like Word, Excel, PowerPoint, and OneNote. You'll get guidance on the various deployment methods that include easy self-install options to enterprise deployments with management tools. The instructions will help you assess your environment, figure out your specific deployment requirements, and implement the necessary support tools to ensure a successful install.
+The [Microsoft 365 Apps setup guide](https://aka.ms/OPPquickstartguide) helps you get your users' devices running the latest version of Office products like Word, Excel, PowerPoint, and OneNote. You'll get guidance on the various deployment methods that include easy self-install options to enterprise deployments with management tools. The instructions will help you assess your environment, figure out your specific deployment requirements, and implement the necessary support tools to ensure a successful install.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
The [Microsoft 365 Apps setup guide](https://aka.ms/OPPquickstartguide) helps you get your users' devices running the latest version of Office products like Word, Excel, PowerPoint, and OneNote. You'll get guidance on the various deployment methods that include easy self-install options to enterprise deployments with management tools. The instructions will help you assess your environment, figure out your specific deployment requirements, and implement the necessary support tools to ensure a successful installation.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**